config/initializers/raven.rb

```diff
@@ -0,0 +1,5 @@
+require 'raven'
```

Comment:
I would restrict usage of this gem only to staging and production environment. It is tedious to be spammed by errors in development, also your collaborators would need to have this set and that is not good. Use the control flow you have learned in `config/initializers/carrierwave.rb` to drive this. Also in gemfile move gem to its group :staging and :production.